WordPress/Classes/Services/OnePasswordFacade.h

```diff
@@ -24,7 +32,25 @@ typedef void (^OnePasswordFacadeCallback)(NSString *username, NSString *password
  */
 - (BOOL)isOnePasswordEnabled;
 
+/**
+ *  Stores a new entry in the 1Password extension.
+ *
+ *  @param URLString                    the URL of the site in question.
+ *  @param loginDetailsDictionary       a dictionary containing login credentials to store.
+ *  @param passwordGenerationOptions    a dictionary specifying password generation options.
+ *  @param viewController the view controller of the class that needs the 1Password extension to appear. Note this can't be nil.
+ *  @param sender         the control that triggered the action
+ *  @param completion     block that is called when 1Password is done setting the credentials.
+ */
+- (void)storeLoginForURLString:(NSString *)URLString
+                  loginDetails:(NSDictionary *)loginDetailsDictionary
+     passwordGenerationOptions:(NSDictionary *)passwordGenerationOptions
+             forViewController:(UIViewController *)viewController
+                        sender:(id)sender
+                    completion:(void (^)(NSDictionary *loginDictionary, NSError * _Nullable error))completion;
```

Comment:
Since this is a _facade_, I'd argue that these changes expose too much of the internal details. I'd propose a simpler API and handling all the details inside the facade:

```obj-c
- (void)createLoginForURLString:(NSString *)URLString
                       username:(NSString *)username
                       password:(NSString *)password
                         sender:(id)sender
                     completion:(void (^)(NSString *username, NSString *password, NSError * _Nullable error))completion;
```